java/src/jmri/beans/PreferencesBean.java

```diff
@@ -94,6 +96,7 @@ protected void setIsDirty(boolean value) {
      * As a side effect, calls to {@link #isDirty} will return {@code true} if
      * oldValue and newValue differ or are null.
      */
+    @SuppressWarnings("javadoc")
```

Comment:
What's the purpose of this suppression? What does it suppress?
Maybe we should lower the JavaDoc warnings instead, if possible.

@bobjacobsen What's your thoughts about this?

Reply:
Our normal Javadoc settings don't require a Javadoc comment on very method, but if there is a Javadoc comment, it needs to have complete `@param`, `@returns`, etc lines in it.  I think that's a reasonable compromise, though sometimes it makes the Javadoc a little verbose when parameters are self-explanatory like "x" or "isDirty".